playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
Note: the values for `/usr/local/bin` and `/tmp` are the default, could also be configured if desired.

The more comprehensive configuration would look like:

```yaml
vars:
  openshift_check_min_host_disk_gb:
    "/var":
      masters: 0.1
      nodes: 1
      etcd: 2
    "/usr/local/bin":
      masters: 0.1
      nodes: 1
      etcd: 2
```

Replacing `0.1`, `1`, `2` with any desired value.

(/cc @sosiouxme, now I realize that configuring the value for the temp dir depends on the value that `tempfile.gettempdir()` evaluates to, which is not really good UX)